src/test/compile-fail/issue-39388.rs

```diff
@@ -11,7 +11,7 @@
 #![allow(unused_macros)]
 
 macro_rules! assign {
-    (($($a:tt)*) = ($($b:tt))*) => { //~ ERROR expected `*` or `+`
+    (($($a:tt)*) = ($($b:tt))*) => { //~ ERROR 14:22: 14:29: expected one of: `*`, `+`, or `?`
```

Comment:
Please move this to a `ui` test instead of hard-coding spans and things (i.e., the `14:22`)

Reply:
Oh, I intended to remove the line/col numbers but forgot... Should I still move to a `ui` test if I remove them?

Reply:
Doesn't matter so much. I believe `ui` tests are generally "the future" but this test can stay a compile-fail test.